tests/test_arrow.py

```diff
@@ -2032,6 +2032,49 @@ def test_multiple_granularity(self):
             == "a minute and 2 seconds ago"
         )
 
+    def test_all_granularity(self):
+        assert (
+            self.now.humanize(granularity="all")
+            == "in 0 years 0 months 0 weeks 0 days 0 hours 0 minutes and 0 seconds"
```

Comment:
Could this cause an issue by not having the `other` parameter set like the rest of the tests?

Reply:
I'm not sure. It might technically be time-sensitive, but in practice it probably won't take >= 1 second. For what it's worth, this other test does essentially the same thing:

https://github.com/arrow-py/arrow/blob/89f77b15a27d57db6ff0b8bf013decf3d970574e/tests/test_arrow.py#L1965-L1967

Reply:
I fixed this anyway in 804eece.